datarepo-clienttests/src/main/java/deploymentscripts/LaunchLocalProcess.java

```diff
@@ -11,12 +11,16 @@
 import java.util.List;
 import java.util.Map;
 import java.util.concurrent.TimeUnit;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import runner.DeploymentScript;
 import runner.config.ApplicationSpecification;
 import runner.config.ServerSpecification;
 import utils.ProcessUtils;
 
 public class LaunchLocalProcess extends DeploymentScript {
+  private static final Logger LOG = LoggerFactory.getLogger(LaunchLocalProcess.class);
```

Comment:
Why up case? That isn't our usual coding convention

Reply:
Good question. I think because this was the convention at my last job and I just didn't think about it. Will update LOG -> logger, to conform with Data Repo.